build-managed.cmd

```diff
@@ -0,0 +1,2 @@
+@call %~dp0run.cmd build-managed -nodoReuse -binclashWindows %*
```

Comment:
Each command needs one tool to run. Build-managed and Build native used different tools so we divided them into different commands.
The developers are not going to call `run build-native` but will call `build.cm/sh` which is what we want.
If they want to be more specific and do special things for only one type of build, they have the possibility.